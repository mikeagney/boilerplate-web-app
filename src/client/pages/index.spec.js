import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Pages from './index';

describe('Page route table component', () => {
  it('will render the pages', () => {
    // Arrange
    const wrapper = shallow(<Pages />);

    // Act
    // Assert
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
