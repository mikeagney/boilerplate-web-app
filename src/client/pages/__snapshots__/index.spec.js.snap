// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page route table component will render the pages 1`] = `
<Switch>
  <Route
    exact={true}
    key="root"
    path="/"
    render={[Function]}
  />
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "load": [Function],
        "preload": [Function],
        "render": [Function],
      }
    }
    exact={true}
    key="characters"
    path="/characters"
  />
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "load": [Function],
        "preload": [Function],
        "render": [Function],
      }
    }
    exact={true}
    key="gauntlet"
    path="/gauntlet"
  />
</Switch>
`;
